### PR TITLE
Extracted Examples into their own README files. Closes #71

### LINKED ISSUE #71: Examples: split examples out of the README to their own directory/README
We're planning more examples, can't really cram them all in the README. We should split them into their own directory so we can provide example files and attach screenshots when necessary.

The main README should have the list of examples and link to them for discoverability.

### PR DESCRIPTION
**Pull request by Skarlso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,101 +136,7 @@ available parameters are in [the reference documentation][pkg-config].
...
-In case of multiple machines the port will be different on each machine.
-
-```console
-$ docker port cluster-node1 80
-0.0.0.0:32828
-
-$ docker port cluster-node0 80
-0.0.0.0:32826
-```
+To see some of the use cases, please refer to the Example's [README](./examples/README.md).
```
Comment (dlespiau) [inline README.md]: It'd be nice to have a list of examples here with links to their README files:

- [Customize the OS image](examples/fedora29-htop/README.md)
- [Expose services](...)
- [Provision the VM with ansible](...)
- ....

Comment (Skarlso) [reply README.md]: Yeah, I was wondering about which would look better.

Comment (Skarlso) [reply README.md]: so you think the extra indirection is too much? Just link from here to the READMEs directly?

**Comment by Skarlso** [comment]
\o/

**Pull request merged**